tests/test_common/test_model/test_declarative/test_entities.py

```diff
@@ -0,0 +1,34 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+import unittest
+from uuid import (
+    UUID,
+    uuid4,
+)
+
+from minos.common import (
+    DeclarativeModel,
+    Entity,
+)
+
+NULL_UUID = UUID("00000000-0000-0000-0000-000000000000")
```

Comment:
You can directly use the `minos.common.NULL_UUID` instead of redefining again here.